src/main/kotlin/no/nav/familie/ef/sak/api/beregning/VedtakDto.kt

```diff
@@ -17,6 +18,13 @@ enum class ResultatType {
     AVSLÅ,
     HENLEGGE
 }
+
+fun ResultatType.tilVedtaksresultat(): Vedtaksresultat = when(this) {
+    ResultatType.INNVILGE -> Vedtaksresultat.INNVILGET // TODO: Når skal vi ha delvis innvilget og opphørt
+    ResultatType.HENLEGGE -> error("Vedtaksresultat kan ikke være henlegge")
```

Comment:
Burde sånt sjekkes før runtime ?

Reply:
Er mest for å please kotlin - må ta høyde for alle enums i en when-mapping